Transfer chapter 4 (mitigating uncertainty)

zslade opened the pull request:
A big chapter containing a lot of social research content so hopefully I've copied over everything sensibly!

zslade committed:
Updating chapter 4

zslade committed:
Updated sections

zslade committed:
Added hyperlinks

zslade committed:
Updated file names

zslade committed:
renaming and reordering

zslade committed:
Fixing image

zslade committed:
Final section of mitigating uncertainty chapter

ghpickering commented:
Looks good - just a couple of small things to consider:

- Probably need to think about whether we should have links on the "landing" section for each chapter (i.e. we do it for all of them or not). I'm leaning towards not including them (and maybe instead writing some more introductory text) but happy to discuss as a group. 
- In "Mitigating uncertainty in qualitative research" could we perhaps tweak the second paragraph slightly? We have that introductory line saying "firstly..." but then I don't think we then go on to introduce validation and transparency in the same way later on so it looks slightly off. Appreciate I might be completely wrong!

zslade committed:
update link

zslade committed:
update chapter name and introduction

zslade committed:
remove links and expand paragraph

zslade commented:
I've removed the links and updated the intro paragraph on `_index.md`. I found it a bit tricky to know what were the most relevant areas to mentioned but hopefully what I've added seems reasonable and representative of what's contained within the subsections of chapter 4

ghpickering reviewed (approved):
Looks good!
